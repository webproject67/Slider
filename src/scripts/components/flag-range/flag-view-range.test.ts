import SliderModel from '../model/slider-model';
import FlagViewRange from './flag-view-range';

const model = new SliderModel();
const flag = new FlagViewRange(model);

test('spyOn mouseDown flag range', () => {
  const somethingSpy = jest.spyOn(flag, 'flagMouseDown');
  let evt: any;
  flag.flagMouseDown(evt);
  expect(somethingSpy).toHaveBeenCalledTimes(1)
});

it('jest snapshots', () => {
  expect(flag.element).toMatchSnapshot()
})