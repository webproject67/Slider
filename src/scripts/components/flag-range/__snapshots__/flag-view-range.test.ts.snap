// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest snapshots 1`] = `
<div
  class="slider__flags"
>
  
    
  <span
    class="slider__flag slider__flag--min"
    style="left:0%"
  >
    0
  </span>
  
    
  <span
    class="slider__flag slider__flag--max"
    data-testid="flagHorizontalRange"
    style="left:100%"
  >
    100
  </span>
  
    
</div>
`;
